decidim-core/lib/decidim/participatory_space_manifest.rb

```diff
@@ -17,6 +17,13 @@ class ParticipatorySpaceManifest
 
     attribute :name, Symbol
 
+    # The ActiveRecord class name of the model we're exposing
+    attribute :model_class_name, String
+
+    # The name of the named Rails route to create the url to the resource.
+    # When not explicitly set, it will use the model name.
+    attribute :route_name, String
+
```

Comment:
Seems unused?

Reply:
Nope, it is used by the `ResourceLocatorPresenter` to find the method name for the path:

https://github.com/decidim/decidim/blob/52f619c7e3167917b32d84f4e4a30c26169b7854/decidim-core/app/presenters/decidim/resource_locator_presenter.rb#L63-L65

https://github.com/decidim/decidim/blob/52f619c7e3167917b32d84f4e4a30c26169b7854/decidim-core/app/presenters/decidim/resource_locator_presenter.rb#L44-L46

https://github.com/decidim/decidim/blob/52f619c7e3167917b32d84f4e4a30c26169b7854/decidim-core/app/presenters/decidim/resource_locator_presenter.rb#L16-L18

I had to add this so the ResourceLocatorPresenter could work with Participatory Spaces as well.

Reply:
Aha! Perfect explanation through 3 little snippets! Thanks @mrcasals! :heart:

Reply:
💪